src/stateful-core.ts

```diff
@@ -44,8 +45,14 @@ export default class StatefulCore {
     this.stateManager.dispatchEvent('vertical/setKey', key);
   }
 
-  setLimit(limit: number): void {
-    this.stateManager.dispatchEvent('vertical/setLimit', limit);
+  setLimit(limit: number | UniversalLimit): void {
```

Comment:
We might just want a separate action for setting the universal limit? The user will probably have to have some switching logic even if we have this internal if statement, since the parameter types are different, so I'm not sure if this makes people's lives easier.

Reply:
I agree. I'll check with Tom on this